resources/nats-streaming/check-namespace.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+seconds=240    #seconds to wait till "knative-eventing namespace" will be created by another Helm 3 installer
+counter=0
+until [ $counter -gt $seconds ]; do
+  timestamp=`date`
+  if kubectl get ns | grep knative-eventing; then
+    echo "[$timestamp] knative-eventing found"
+    break
+  else
+    echo "[$timestamp] knative-eventing not found"
+  fi
+  sleep 1
+  ((counter++))
```

Comment:
is it really worth checking that often?

Reply:
Change to wait for 5s before checking again